bundles/org.openhab.binding.senechome/src/main/java/org/openhab/binding/senechome/internal/SenecHomeApi.java

```diff
@@ -71,17 +69,26 @@ public void setHostname(String hostname) {
      */
     public SenecHomeResponse getStatistics()
             throws InterruptedException, TimeoutException, ExecutionException, IOException {
-        String location = HTTP_PROTO_PREFIX + hostname;
+        String location = hostname + "/lala.cgi";
+        logger.trace("sending request to: {}", location);
+
+        try {
+            httpClient.start();
+        } catch (Exception e) {
+            logger.error("cannot connect", e);
+        }
 
         Request request = httpClient.newRequest(location);
         request.header(HttpHeader.ACCEPT, MimeTypes.Type.APPLICATION_JSON.asString());
-        request.header(HttpHeader.CONTENT_TYPE, MimeTypes.Type.FORM_ENCODED.asString());
+        request.header(HttpHeader.CONTENT_TYPE, MimeTypes.Type.APPLICATION_JSON.asString());
         ContentResponse response = null;
         try {
-            response = request.method(HttpMethod.POST)
-                    .content(new StringContentProvider(gson.toJson(new SenecHomeResponse()))).send();
+            String dataToSend = gson.toJson(new SenecHomeResponse());
+            logger.trace("data to send: {}", dataToSend);
+            response = request.method(HttpMethod.POST).content(new StringContentProvider(dataToSend)).send();
```

Comment:
Please fix either the javadoc or the catch block because the method can currently return a lot more exceptions than IOException.

Reply:
updated javadoc